lib/htmx_rails.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```

Comment:
maybe this file can be `htmx-rails.rb` for consistency? `-` and `_` have different meanings

Reply:
You're right. So we'll have `lib/htmx-rails.rb` and `lib/htmx/rails/version.rb`. I will delete Base module and put its content in `lib/htmx-rails.rb`. Is this ok?

Reply:
Sounds great to me! Just in case we already did something similar here https://github.com/rootstrap/activeadmin-chat/tree/master/lib and I think `react-rails` did the same

Reply:
Nice! And what about [this file](https://github.com/rootstrap/htmx-rails/blob/install-generators/spec/htmx_rails/base_spec.rb)? I'm not sure of its location, or even if we should keep it

Reply:
haha yea it's a weird case, I guess it's not really necessary